### DISCUSSION TITLE
Opencanary should not block on log destination not available

### DISCUSSION THREAD

**Discussion by thinkst-pieter** [thinkst/opencanary#343] [Feature Requests]
@SecuriLee [reported](https://github.com/thinkst/opencanary/issues/342) difficulties in having OpenCanary start because of a log destination not being reachable.

Ideally you would want a clear indication that a log destination is not available and not have it pause the OpenCanary service from coming up.

**Comment by SecuriLee** [comment]
indeed if a disk is full or a destination is unavailable, maybe there needs to be a CI that allows the OC to report this to the console and continue.  Examples: webhook destination *gone*, disk full (because you're nuts enough to have an open SMB share to the Internet, ....)

And no worries about response times, I see no SLA here ;)
